stream/src/public_key/connection.rs

```diff
@@ -168,47 +187,54 @@ impl<Si: Sink, St: Stream> Connection<Si, St> {
     /// Because we already verified the peer's handshake, this function
     /// only needs to send our handshake message for the connection to be fully
     /// initialized.
-    pub async fn upgrade_listener<R: Rng + CryptoRng + Spawner + Clock, C: Scheme>(
+    pub async fn upgrade_as_dialee<R: Rng + CryptoRng + Spawner + Clock, C: Scheme>(
```

Comment:
nit: I'm definitely biased, but I strongly prefer `upgrade_listener` and `upgrade_dialer` to the new names

Reply:
I can remove the `as` part.

However, everywhere else we refer to the peer being dialed as the "dialee" and nowhere else do we call it the "listener". We should either call it `upgrade_dialee` or change all `dialee` references to `listener` IMO. Given that, it seems using `dialee` is better

Reply:
I'd strongly prefer `dialer`/`listener` everywhere. I think that makes it much clearer what is going on.

Reply:
Changed